apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -35,6 +37,13 @@ public final class TestOutputActivity extends TestOutputActivityBase {
     private CheckBox[] mChannelBoxes;
     private Spinner mOutputSignalSpinner;
 
+    public static final String KEY_SIGNAL_TYPE = "signal_type";
```

Comment:
Move KEY VALUE to IntentBasedtestSupport.java

Reply:
Done